### PR TITLE
Chapter 6 - Modeling users

### PR DESCRIPTION
**Pull request by linhheo**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class User < ApplicationRecord
+  # chuyển email về dạng chữ in thường
```
Comment (b-cuong) [inline app/models/user.rb]: Remove ALL comments in this file. Make a note or something that out of your project to keep your knowledges

File: test/fixtures/users.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```
Comment (b-cuong) [inline test/fixtures/users.yml]: Remove auto-generated comment

File: test/models/user_test.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class UserTest < ActiveSupport::TestCase
+  # test "the truth" do
```
Comment (b-cuong) [inline test/models/user_test.rb]: Remove auto-generated comment

File: app/models/user.rb (hunk 1 above)
Comment (linhheo) [reply app/models/user.rb]: Dạ vâng anh

**Pull request merged**